src/components/Form/Controls/Select.tsx

```diff
@@ -46,6 +41,13 @@ export class Select extends Base<Props> {
   @Model('current value', { event: 'change', type: [String, Number, Object] })
   public value!: Value;
   private currentValue = this.value;
+  @Inject({ from: FORM_ITEM_KEY, default: null}) public formItem!: FormItem | null;
+
+  private get uid(): string {
+    const item = this.formItem;
+    if(item == null) { return ''; }
```

Comment:
Just curious: why we check with `if(item == null)` and not with `if(!item)` ? 
`if(!item)` will check for empty strings, null and undefined

Reply:
That is a good question.

The ivar `formItem` is statically typed and provided by the `FormItem`-component. It can never be a `string` (including the empty string) or `undefined`. Well it can if the type annotation is done incorrectly in the first place or if `FormItem` breaks the implicitly established contract ("I am always providing a value of type `FormItem`"). But then this more like a programmer error and we should not even continue execution but throw an error and simply fix the problem.

Does that make sense?